custom/conf/app.ini.sample

```diff
@@ -327,6 +327,8 @@ MIN_PASSWORD_LENGTH = 6
 IMPORT_LOCAL_PATHS = false
 ; Set to true to prevent all users (including admin) from creating custom git hooks
 DISABLE_GIT_HOOKS = false
+;Set character types for password complexity. Lack or misconfiguration disables the complexity check requirement
+PASSWORD_COMPLEXITY = lower,upper,digit,spec
```

Comment:
This should be the default value but it seems they are not.

Reply:
Hello!

I think this is a slight misunderstanding
on line 774 in the setting.go file a default set of parameters is defined

![image](https://user-images.githubusercontent.com/19513404/64941715-9436fd00-d891-11e9-8abd-735437e6d5e5.png)